pkg/validate/types.go

```diff
@@ -0,0 +1,67 @@
+package validate
```

Comment:
BTW I think we need to unify the package naming. We have manifest validation inside `manifest` package as far as I remember - maybe we should have Action validation in some `action` one? To discuss

Reply:
after f2f discussion our approach is:
- `pkg/validation/action`
- `pkg/validation/manifest`
- `pkg/validation/` - common stuff